src/lecturer/components/studentsQuestionView/StudentsQuestionListView.tsx

```diff
@@ -64,28 +84,47 @@ export function StudentsQuestionListView() {
         };
     }, [refreshQuestionList, socketEmiter]);
 
-
     return (
         <Paper className={classes.root} variant="outlined" square>
+        <b className={classes.questionsHeader}>
+            Pytania od studentów:
+        </b>
+        <div className={classes.questionField}>
+            <ReactScrollableFeed>
             {store.studentQuestions.map((studentQuestion, index) => {
                 return (
                     <div
                         key={index}
                     >
                         <div className={classes.message}>
                             <div className={classes.messageText}>
-                                <b className={classes.messageHeader}>
-                                    {studentQuestion.time.toLocaleTimeString("en-GB") + " | Anonimowy student"}:
-                                </b>
-                                <div className="question-text">
-                                    {studentQuestion.text}
-                                </div>
+                            {studentQuestion.processed ? (
+                                    <TextField className={classes.field} variant="outlined" fullWidth={true}
+                                    multiline
+                                    label={studentQuestion.time.toLocaleTimeString("en-GB") + " | Anonimowy student"} 
+                                    defaultValue={studentQuestion.text} 
+                                    InputProps={{
+                                        className: classes.questionText,
+                                        readOnly: true,
+                                    }} />                            
+                            ):(
+                                <TextField className={classes.field} error variant="outlined"
+                                 fullWidth={true} 
+                                 multiline
+                                 label={studentQuestion.time.toLocaleTimeString("en-GB") + " | Anonimowy student"} 
+                                 defaultValue={studentQuestion.text} InputProps={{
+                                    className: classes.questionText,
+                                    readOnly: true,
+                                }}/>
+                            )}
```

Comment:
```suggestion
                                <TextField className={classes.field} error={!studentQuestion.processed} variant="outlined"
                                 fullWidth={true} 
                                 multiline
                                 label={studentQuestion.time.toLocaleTimeString("en-GB") + " | Anonimowy student"} 
                                 defaultValue={studentQuestion.text} InputProps={{
                                    className: classes.questionText,
                                    readOnly: true,
                                }}/>
```
You could remove this duplicating code by moving "if" statement into the  "error" property